docs/assets/versions.js

```diff
@@ -80,11 +80,20 @@ function sortVersionLinks(container) {
   const dlElements = container.querySelectorAll('dl');
 
   dlElements.forEach(dl => {
-    const dt = dl.querySelector('dt');
-    if (dt && dt.textContent.trim().toLowerCase() === 'versions') {
-      // Found the Versions <dl>
-      const ddElements = Array.from(dl.querySelectorAll('dd'));
+    const ddElements = Array.from(dl.querySelectorAll('dd'));
+
+    // Check if ddElements contain version links
+    const isVersionDl = ddElements.some(dd => {
+      const link = dd.querySelector('a');
+      if (link) {
+        // Check if the link's href matches version pattern
+        return /\/en\/(release-(?:v\d+|[\d\.]+|\w+)|latest|stable)\//.test(link.getAttribute('href'));
+      }
+      return false;
```

Comment:
```suggestion
      return /\/en\/(release-(?:v\d+|[\d\.]+|\w+)|latest|stable)\//.test(link?.getAttribute?.('href'));
```
This could be simplified.

 If the version regex expression is the same as above, maybe we should store it in a constant.

Reply:
I used a constant for the regex as you suggested. Thanks!